charts/cloudzero-agent/configuration.example.yaml

```diff
@@ -4,12 +4,10 @@ cloudAccountId: null
 clusterName: null
 # -- Region the cluster is running in.
 region: null
-
-global:
-  # -- CloudZero API key. Required if useExistingSecret is false.
-  apiKey: null
-  # -- If set, the agent will use the API key in this Secret to authenticate with CloudZero.
-  existingSecretName: null
+# -- CloudZero API key. Required if existingSecretName is false.
```

Comment:
Should this say `null` instead of `false`?

Reply:
yes, fixed, thanks!